pkgs/by-name/ta/tauon/package.nix

```diff
@@ -144,9 +138,9 @@ stdenv.mkDerivation (finalAttrs: {
     description = "Linux desktop music player from the future";
     mainProgram = "tauon";
     homepage = "https://tauonmusicbox.rocks/";
-    changelog = "https://github.com/Taiko2k/TauonMusicBox/releases/tag/v${finalAttrs.version}";
+    changelog = "https://github.com/Taiko2k/TauonMusicBox/releases/tag/v${version}";
```

Comment:
The `TauonMusicBox` is now `Tauon`, I still have no idea why developer did this, probably doesn't matter in this scope since github redirects it correctly anyway